Consul/Operator.cs

```diff
@@ -100,6 +100,37 @@ public class KeyringResponse
         public int NumNodes { get; set; }
     }
 
+    public class AreaRequest
+    {
+        /// <summary>
+        /// PeerDatacenter is the peer Consul datacenter that will make up the
+        /// other side of this network area. Network areas always involve a pair
+        /// of datacenters: the datacenter where the area was created, and the
+        /// peer datacenter. This is required.
+        /// </summary>
+        public string PeerDatacenter { get; set; }
+
+        /// <summary>
+        /// RetryJoin specifies the address of Consul servers to join to, such as
+	    /// an IPs or hostnames with an optional port number. This is optional.
+        /// </summary>
+        public string[] RetryJoin { get; set; }
+
+        /// <summary>
+        /// UseTLS specifies whether gossip over this area should be encrypted with TLS
+        /// if possible.
+        /// </summary>
+        public bool UseTLS { get; set; }
+    }
+
+    public class Area : AreaRequest
+    {
+        /// <summary>
+        /// ID is this identifier for an area (a UUID). This must be left empty
```

Comment:
With our API, the comment `This must be left empty when creating a new area.` is not relevant.